README.md

```diff
@@ -42,30 +42,92 @@ Before pushing the site to production, and a great way to demonstrate proposed
 changes during code review, you should push the changes to your personal GitHub
 fork as follows:
 
-1. fork the main repo https://github.com/JanusGraph/janusgraph.org to your
-   account
+1. [Fork the website repo](https://github.com/JanusGraph/janusgraph.org#fork-destination-box)
+   to your account via GitHub
    * note: the `master` branch of this repo is the one that is auto-published to
      http://janusgraph.org soon after a commit
-1. clone your copy of the repo locally
-1. checkout the `gh-pages` branch or create it if it doesn't exist
-1. make changes on that branch and push `gh-pages` to your fork on GitHub
-1. visit the "Settings" page for your fork on GitHub and make the `gh-pages`
+
+1. Clone your copy of the repo locally
+
+   ```
+   $ git clone git@github.com:[USERNAME]/janusgraph.org.git
+   $ cd janusgraph.org
+   $ git remote add upstream git@github.com:JanusGraph/janusgraph.org.git
+   ```
+
+1. Create a new branch for your changes, based on the latest changes in the
+   `master` branch of the upstream repo:
+
+   ```
+   $ git checkout master
+   $ git pull upstream master
```

Comment:
Assuming my clone is old one, we should also do `git merge upstream/master` to make sure local `master` has latest from the upstream repo.